R/lazyframe-frame.R

```diff
@@ -270,6 +587,9 @@ lazyframe__sort <- function(
     check_dots_unnamed()
 
     by <- parse_into_list_of_expressions(...)
+    if (length(by) == 0) {
+      abort("`...` must contain at least one element.")
+    }
```

Comment:
In py-polars, sort() on empty errors because it requires a positional argument but here, without this condition it returns the input.

Reply:
Perhaps `!missing(...)` would be more appropriate?